src/components/Filters.vue

```diff
@@ -151,21 +126,81 @@ import { closeCircleOutline, businessOutline, gitBranchOutline, gitPullRequestOu
 import { translate } from '@/i18n'
 import store from "@/store";
 import router from "@/router";
+import { DateTime } from "luxon";
 
 const dateTimeModalOpen = ref(false)
+const currentDateFilter = ref("");
+const currentDateFilterValue = ref("") as any;
 
 const facilities = computed(() => store.getters["user/getFacilities"])
 const query = computed(() => store.getters["count/getQuery"])
 
-function openDateTimeModal() {
+function openDateTimeModal(dateFilterKey: string) {
+  currentDateFilter.value = dateFilterKey;
+  currentDateFilterValue.value = query.value[dateFilterKey] ? query.value[dateFilterKey] : DateTime.now();
   dateTimeModalOpen.value = true;
 }
 
+// Returns the minimum allowed date for the current date filter, ensuring it is not before the "from" date for closed or created dates.
+function getMinDate() {
+  const dateFilterKey = currentDateFilter.value;
+
+  if(dateFilterKey === 'closedDate_thru') {
+    const afterDateClosed = query.value.closedDate_from;
+    return afterDateClosed ? DateTime.fromISO(afterDateClosed).toISO() : undefined;
+  } else if(dateFilterKey === 'createdDate_thru') {
```

Comment:
I think we should add date cross checks as well, like if we have selected createdAfter 1st Jan, then not allow selecting closed date before 1st Jan.